components/layout/header/index.tsx

```diff
@@ -70,4 +68,50 @@ const HeaderCS: React.FC = () => {
   );
 };
 
+const ManageSession = () => {
+  const { data: session } = useSession();
+  if (session && session.user) {
+    return <UserProfileButton user={session.user} />;
+  } else {
+    return <SignInButton />;
+  }
+};
+
+const SignInButton = () => {
+  return (
+    <S.HeaderButton
+      size={ButtonSize.Small}
+      onClick={() => signIn("slack")}
+      inverted
+    >
+      Přihlásit
+    </S.HeaderButton>
+  );
+};
+
+const UserProfileButton: React.FC<{
+  user: NonNullable<DefaultSession["user"]>;
+}> = ({ user }) => {
+  const imageUrl =
+    user.image || "https://data.cesko.digital/people/generic-profile.jpg";
+  const email = user.email || "<neznámý e-mail>";
+  return (
+    /* eslint-disable @next/next/no-img-element */
+    <img
```

Comment:
Na tohle by chtělo nějakou komponentu `<Avatar />`?

Reply:
Já jsem ji i napsal, ale dokud je to čistě lokální helper bez dalších volajících, tak to je IMO vlastně akorát víc kódu bez reálného užitku. Čili bych nechal takhle a pokud to začneme používat na více místech, můžeme zobecnit?